taxcalc/policy.py

```diff
@@ -40,9 +40,9 @@ class instance: Policy
     DEFAULTS_FILE_NAME = 'policy_current_law.json'
     DEFAULTS_FILE_PATH = os.path.abspath(os.path.dirname(__file__))
     JSON_START_YEAR = 2013  # remains the same unless earlier data added
-    LAST_KNOWN_YEAR = 2025  # last year for which indexed param vals are known
+    LAST_KNOWN_YEAR = 2026  # last year for which indexed param vals are known
```

Comment:
Should the `LAST_KNOWN_YEAR` stay at 2025?  This PR doesn't update with IRS rounding rules.